scripts/build_binary.sh

```diff
@@ -19,15 +19,7 @@ cd "${ROOT}"
 # Builds the ecs-cli binary from source in the specified destination paths.
 mkdir -p $1
 
-# Versioning stuff. We run the generator to setup the version and then always
-# restore ourselves to a clean state
-cp ecs-cli/modules/version/version.go ecs-cli/modules/version/_version.go
-trap "cd \"${ROOT}\"; mv ecs-cli/modules/version/_version.go ecs-cli/modules/version/version.go" EXIT SIGHUP SIGINT SIGTERM
-
-cd ./ecs-cli/modules/version/
-go run gen/version-gen.go
-
-cd "${ROOT}"
-
-GOOS=$TARGET_GOOS CGO_ENABLED=0 go build -installsuffix cgo -a -ldflags '-s' -o $1/ecs-cli ./ecs-cli/
+GIT_DIRTY=`git diff --quiet || echo '*'`
```

Comment:
Do we want to use `git describe tag` here like we do in the [v2 version command](https://github.com/aws/amazon-ecs-cli-v2/pull/212/files#diff-ab45564d4a3a96a4af2f6455a2b3613eR9)?

Reply:
We definitely can but it will change the output format of v1's version command. So I want to avoid that.

Reply:
Update: Will try to modify the version command in v1 to only output the git hash when the version string is empty